packages/open-payments/src/client/index.ts

```diff
@@ -258,9 +288,37 @@ export interface AuthenticatedClient
   quote: QuoteRoutes
 }
 
-export const createAuthenticatedClient = async (
+/**
+ * @experimental The `requestInterceptor` feature is currently experimental and might be removed in upcoming versions. Use at your own risk! It offers the capability to add a custom method for generating HTTP signatures. It is recommended to create the authenticated client with the `privateKey` and `keyId` arguments. If both `requestInterceptor` and `privateKey`/`keyId` are provided, an error will be thrown.
+ * @throws OpenPaymentsClientError
+ */
+export async function createAuthenticatedClient(
+  args: CreateExperimentalAuthenticatedClientArgs
+): Promise<AuthenticatedClient>
+/**
+ * JSDoc comment here
+ * @throws OpenPaymentsClientError
```

Comment:
"Creates an Open Payments client that exposes methods to call all of the Open Payments APIs. Each request requiring authentication will be signed with the given private key."

Reply:
Let's move the "normal" `createAuthenticatedClient` signature first. It'll be the first JSDoc autocomplete result when someone types `createAuthenticatedClient`